### PR TITLE
Update readme

### PR DESCRIPTION
**Pull request by jin-s13**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -22,12 +22,66 @@ The master branch works with **PyTorch 1.3+**.
   <img src="demo/demo.gif" width="600px"/>
 </div>
 
+
+### Major Features
+
+- **Modular design**
+
```
Comment (innerlee) [inline README.md:29]: move this item to the bottom since it is quite standard now

File: README.md [hunk 2]
```diff
@@ -22,12 +22,66 @@ The master branch works with **PyTorch 1.3+**.
...
+- [x] [Hourglass](configs/top_down/hourglass/README.md)
+- [x] [SCNet](configs/top_down/scnet/README.md)
+- [x] [HigherHRNet](configs/bottom_up/higherhrnet/README.md)
+- [x] [DarkPose](configs/top_down/darkpose/README.md)
 
 Results and models are available in the *README.md* of each method's config directory.
 A summary can be found in the [**model zoo**](https://mmpose.readthedocs.io/en/latest/model_zoo.html) page.
+We will keep up with the latest progress of the community, and support more popular algorithms and frameworks.
+
+### TODO List
```
Comment (innerlee) [inline README.md:79]: how about remove this section, and replace it with link and description of that pinned issue

File: README.md (hunk 1 above)
Comment (jin-s13) [reply README.md:29]: OK.

File: README.md (hunk 2 above)
Comment (jin-s13) [reply README.md:79]: OK.

**Pull request merged**